Composer/packages/types/src/shell.ts

```diff
@@ -70,6 +71,8 @@ export type ApplicationContextApi = {
   isFeatureEnabled: (featureFlagKey: FeatureFlagKey) => boolean;
   setApplicationLevelError: (err: any) => void;
   confirm: (title: string, subTitle: string, settings?: any) => Promise<boolean>;
+  // TODO: Add TelemetryClient interface
```

Comment:
What do you mean?

Reply:
Already done - forgot to remove the comment.